client/src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+
+import SidebarHead from './SidebarHead'
+import SidebarBody from './SidebarBody'
+
+import "./style.scss"
+
+export enum SIDEBAR_ITEM {
+  USERS = "USERS",
+  CHAT = "CHAT",
+}
+
+const SideBar = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(false)
```

Comment:
Let's start a practice of adding comment descriptions (in a full sentence) to every state.

Reply:
We can just leave this up to the person writing the code as per messenger.